tests/testthat/test-test_visualizations.R

```diff
@@ -0,0 +1,64 @@
+source(file.path(getwd(), "data_helper.R"))
+
+# Test preparation
+######################################################################
+test_data = medium_test_data()
+prediction = simple_rpart_prob_predictions()
+benchmark_obj = simple_benchmark_result()
+resample_obj = simple_resample_result()
+fairness_measure = msr("fairness.acc")
+fairness_measures = msrs(c("fairness.tpr", "fairness.fnr"))
+######################################################################
+
+# fairness_accuracy_tradeoff Tests
+test_that("fairness_accuracy_tradeoff work properly with PredictionClassif", {
+  expect_error(fairness_accuracy_tradeoff(prediction, fairness_measure, test_data), NA)
+  expect_warning(fairness_accuracy_tradeoff(prediction, fairness_measure, test_data), NA)
+})
+
+test_that("fairness_accuracy_tradeoff work properly with ResampleResult", {
+  expect_error(fairness_accuracy_tradeoff(resample_obj, fairness_measure), NA)
+  expect_warning(fairness_accuracy_tradeoff(resample_obj, fairness_measure), NA)
+})
+
+test_that("fairness_accuracy_tradeoff work properly with BenchmarkResult", {
+  expect_error(fairness_accuracy_tradeoff(benchmark_obj, fairness_measure), NA)
+  expect_warning(fairness_accuracy_tradeoff(benchmark_obj, fairness_measure), NA)
+})
+
+# fairness_compare Tests
+test_that("fairness_compare work properly with PredictionClassif", {
+  # Single Measure
+  expect_error(fairness_compare(prediction, fairness_measure, test_data), NA)
+  expect_warning(fairness_compare(prediction, fairness_measure, test_data), NA)
+
+  # Multiple Measures
+  expect_error(fairness_compare(prediction, fairness_measures, test_data), NA)
+  expect_warning(fairness_compare(prediction, fairness_measures, test_data), NA)
+})
+
+test_that("fairness_compare work properly with ResampleResult", {
+  # Single Measure
+  expect_error(fairness_compare(resample_obj, fairness_measure), NA)
+  expect_warning(fairness_compare(resample_obj, fairness_measure), NA)
+
+  # Multiple Measures
+  expect_error(fairness_compare(resample_obj, fairness_measures), NA)
+  expect_warning(fairness_compare(resample_obj, fairness_measures), NA)
+})
+
+test_that("fairness_compare work properly with BenchmarkResult", {
+  # Single Measure
+  expect_error(fairness_compare(benchmark_obj, fairness_measure), NA)
+  expect_warning(fairness_compare(benchmark_obj, fairness_measure), NA)
+
+  # Multiple Measures
+  expect_error(fairness_compare(benchmark_obj, fairness_measures), NA)
+  expect_warning(fairness_compare(benchmark_obj, fairness_measures), NA)
+})
+
```

Comment:
Test for ggplot object etc, see https://github.com/mlr-org/mlr3benchmark/blob/main/tests/testthat/test_autoplot_BenchmarkAggr.R

Reply:
Finished, also add helper function to simplify the structure.